nxlemon/api/maps.pxd

```diff
@@ -0,0 +1,76 @@
+cdef extern from "lemon/concepts/maps.h" namespace "lemon:concepts":
+    cdef cppclass ReadMap[K, T]:
+        ctypedef K Key
+        ctypedef T Value
+
+        Value operator[](Key&)
+
+        cppclass Constraints[_ReadMap]:
+            void constraints()
+
+            Key& key
+
+            #typename _ReadMap::Key& own_key
```

Comment:
All similar lines are commented because it appears that we can't use `typename` like this in Cython. Related - https://groups.google.com/forum/#!topic/cython-users/qQpMo3hGQqI